vault/dynamic_system_view.go

```diff
@@ -87,3 +88,41 @@ func (d dynamicSystemView) ReplicationState() consts.ReplicationState {
 	d.core.clusterParamsLock.RUnlock()
 	return state
 }
+
+// ResponseWrapData wraps the given data in a cubbyhole and returns the
+// token used to unwrap.
+func (d dynamicSystemView) ResponseWrapData(data map[string]interface{}, ttl time.Duration, jwt bool) (string, error) {
+	req := &logical.Request{
+		Operation: logical.CreateOperation,
+		Path:      "sys/init",
```

Comment:
This doesn't feel like the right path to use; I'm not sure if the right approach is to use the request path or something else, but at the very minimum this feels like it should maybe be `sys/wrapping/wrap`.